WordPress/Classes/ViewRelated/Site Creation/Wizard/SiteCreationWizardStepInvoker.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// Responsible for initialization and invocation of WizardSteps
+protocol SiteCreationWizardStepInvoker {
```

Comment:
This was just to give us a way to test without going through the trouble of creating actual steps.

Reply:
That sounds good @Gio2018, let me see if I can improve this.

Reply:
I've pushed up some code that should simplify things.